includes/AppLeave.php

```diff
@@ -120,6 +122,8 @@
 	$qry = $qry." $stat VARCHAR(255) NOT NULL , PRIMARY KEY($AppId) , INDEX idx USING BTREE($AppDate) )";
 	$qry1 = $qry1."\"$status\")";
 	
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->